ext/standard/basic_functions.c

```diff
@@ -1436,6 +1436,11 @@ PHP_FUNCTION(error_get_last)
 
 		ZVAL_LONG(&tmp, PG(last_error_lineno));
 		zend_hash_update(Z_ARR_P(return_value), ZSTR_KNOWN(ZEND_STR_LINE), &tmp);
+
+		if (!Z_ISUNDEF(EG(error_backtrace))) {
+			ZVAL_COPY(&tmp, &EG(error_backtrace));
```

Comment:
Looking at this function context: The global should probably be renamed to `last_error_backtrace` for consistency with the other globals.

Reply:
what about `fatal_error_backtrace`? since it won't be populated with anything other than fatal errors now.

Reply:
I've speculatively went ahead with renaming it `fatal_error_backtrace`, and as a consequence I've renamed the `fatal_error_backtraces` global (not the INI) to `fatal_error_backtrace_on`. Let me know if this change is disagreeable.

My concern with `last_error_backtrace` is that it may not be populated for most "last errors", but I don't know that I feel strongly here.

Reply:
No _strong_ opinion here. `last_fatal_error_backtrace` might be another option to include both the common “last” prefix and the “fatal” indicator. Not a blocker in any case.

Reply:
> `last_fatal_error_backtrace` might be another option to include both the common “last” prefix and the “fatal” indicator. 

done!